packages/request-client.js/package.json

```diff
@@ -49,6 +49,7 @@
     "@requestnetwork/epk-signature": "0.5.7",
     "@requestnetwork/multi-format": "0.2.4",
     "@requestnetwork/request-logic": "0.8.3",
+    "@requestnetwork/smart-contracts": "0.3.0",
```

Comment:
I did some checks to clear my conscience about the size increase of the library.
from `1460818` to `1475340`. It's a lot less than I expected 👍